calculator.js

```diff
@@ -1,5 +1,57 @@
-const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
+const prompt = require('prompt');
+
+const calculateUserInput = function (error, promptInput) {
+  calculate(promptInput);
 }
+
+function validNum(input) {
+  input.num1 = Number(input.num1);
+  input.num2 = Number(input.num2);
+
+  if(isNaN(input.num1 && input.num2)) {
+    console.log("Error: invalid number");
+    return false;
+  } else {
+    return true;
+  }
+}
+
+function determineOperation(input) {
+  let operation = input.operation;
```

Comment:
Since this isn't meant to get updated, it'd be more appropriate to make it `const`